parsely/parselyandroid/ParselyTracker.java

```diff
@@ -210,26 +247,38 @@ public void startEngagement(String url) {
         this.engagementManager.start();
     }
 
-    /*! \brief Add an event Map to the queue.
+    /*! \brief Stop engaged time tracking.
      *
-     *  Place a data structure representing the event into the in-memory queue for later use
-     *
-     *  **Note**: Events placed into this queue will be discarded if the size of the persistent queue
-     *  store exceeds `storageSizeLimit`.
+     * Stops the engaged time tracker, sending any accumulated engaged time to Parse.ly.
+     * NOTE: This *must* be called during various Android lifecycle events like onPause or
+     * onStop. Otherwise, engaged time tracking will keep running and Parse.ly values
+     * may be inaccurate.
      */
-
     public void stopEngagement() {
         if (this.engagementManager == null) {
-            PLog("No ongoing engagement to stop.");
             return;
         }
-        PLog("stopEngagement called");
         this.engagementManager.stop();
         this.engagementManager = null;
     }
 
+    /*! \brief Start video tracking.
+     *
+     * Starts engaged time tracking for a video. Will send a video start event unless the
+     * same video had previously been paused. Video metadata must be provided, specifically
+     * the video ID and video duration.
+     *
+     * The URL value is used for videos embedded in other posts. This is because a single
+     * video can be embedded in multiple articles, but that is more common on full webpages.
+     * Most apps do not embed videos in articles, but instead play them full screen. In that
```

Comment:
I don't think I've seen a mobile browser or app that plays videos in a non-fullscreen view. However, I still feel that having a video fullscreened doesn't preclude it being logically "embedded" in some other content, in this case an app view. I'd rephrase this docstring to indicate that the url can be a real url representing the embedding content or a "real-looking" url representing a dummy page that will make it through the backend's allowed-domains checks.